src/components/FormComponents/PFDivider.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+
+import { Divider } from '@patternfly/react-core';
+import { useFlag } from '@unleash/proxy-client-react';
+
+const PFDivider = () => {
+  const rhelAws = useFlag('platform.sources.metered-rhel');
+
+  return <Fragment>{rhelAws && <Divider />}</Fragment>;
```

Comment:
```jsx
if(!rhelAws) {
  return null
}

return <Divider />
```

Also, I think we might want to make this component slightly generic. But we don't really have custom conditions. You might want to use the divider component even if the flag is false in some other cases. But I think it would need some DDF changes.